### ISSUE TITLE
Some videos getting distorted...

### ISSUE THREAD

**Issue by selimonline** [google/ExoPlayer#4348]
Problems : 
=====================
few videos getting distorted at certain position
here is screenshot.....
https://image.prntscr.com/image/sp6cF6eXQQKwy21J9XkckA.png

Build.gradle 
===============
```
implementation 'com.google.android.exoplayer:exoplayer-core:2.7.0'
implementation 'com.google.android.exoplayer:exoplayer-ui:2.7.0'
```

(although I tried most latest versions). same issues always... 

Activity code : 
=================
```
BandwidthMeter bandwidthMeter = new DefaultBandwidthMeter();
        TrackSelector trackSelector = new DefaultTrackSelector(new AdaptiveTrackSelection.Factory(bandwidthMeter));
        player = ExoPlayerFactory.newSimpleInstance(view.getContext(), trackSelector);


        Uri videoUri = Uri.parse("HTTPS SRC PATH");
        PlayerView video_view = (PlayerView) view.findViewById(R.id.video_view);

        DefaultDataSourceFactory dataSourceFactory = new DefaultDataSourceFactory(this, Util.getUserAgent(this, "oevo"));
        MediaSource videoSource = new ExtractorMediaSource(videoUri,dataSourceFactory, new DefaultExtractorsFactory(), null, null);

player.setPlayWhenReady(true); player.prepare(videoSource);
```

Testing Device : 
Samsung edge s7

Video Url : 
https://oevo.com/exo/TFWsBEnXMgqncUxRxMmGlPAjALAWsqsM.mp4

any idea highly appreciated. Thanks

**Comment by ojw28** [comment]
This appears to be edit list related. I suspect the video falls under the

> Edit lists that require prerolling from a sync sample after reordering are not supported

case in `AtomParsers`. What do we actually do in this case? Should we ignore the edit list, rather than doing whatever we're currently doing? Whatever we're currently doing appears to be what introduces the distortion.

**Comment by andrewlewis** [comment]
Ignoring edit lists for all tracks in this situation is probably preferable to the current behavior for most apps.

Until that is implemented, I suggest setting `DefaultExtractorsFactory.setMp4ExtractorFlags(FLAG_WORKAROUND_IGNORE_EDIT_LISTS)` on the extractors factory to ignore all edit lists. We are unlikely to implement support for prerolling media for edit lists any time soon.

**Comment by ojw28** [comment]
Fixed in `dev-v2`.

**Comment by selimonline** [comment]
Thanks everyone, It seems you have fixed all these issues on version 2.8.4. If still anyone having same issues, make sure you are using version 2.8.4. Exo is best for android 👍 

Thanks again bro @ojw28
